extensions/amp-facebook/validator-amp-facebook.protoascii

```diff
@@ -14,11 +14,31 @@
 # limitations under the license.
 #
 
-tags: {  # amp-facebook
+
+tags: {  # amp-facebook 1.0
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-facebook 1.0"
+  satisfies: "amp-facebook 1.0"
```

Comment:
This would be a good place to leave a comment for the reader on what this scheme is trying to do, maybe even just linking to the discussion in this PR as sufficient. What do you think?

Reply:
Good idea, linked this PR and a brief explanation.Thanks for the recommendation